schema/filter_artworks.js

```diff
@@ -131,14 +132,20 @@ function filterArtworks(primaryKey) {
         type: GraphQLString,
       },
     },
-    resolve: (root, options, request, { rootValue: { accessToken } }) => {
+    resolve: (root, options) => {
+      const gravityOptions = Object.assign({}, options);
       if (primaryKey) {
-        options[primaryKey] = root.id; // eslint-disable-line no-param-reassign
+        gravityOptions[primaryKey] = root.id;
       }
-      if (accessToken) {
```

Comment:
Was this removed by accident? This is causing a bug right now: https://github.com/artsy/force/issues/334 since there are certain filter queries where we do want to pass along the access token if present.